python/llm/dev/benchmark/all-in-one/run.py

```diff
@@ -95,7 +97,7 @@ def run_model(repo_id, test_api, in_out_pairs, local_model_hub=None, warm_up=1,
                             num_beams,
                             low_bit,
                             cpu_embedding if 'win' in test_api else 'N/A',
-                            result[in_out_pair][-1][5] if 'win' in test_api else 'N/A']) # currently only peak mem for win gpu is caught here
+                            result[in_out_pair][-1][5] if 'transformer_int4_gpu' in test_api else 'N/A']) # currently only peak mem for transformer_int4_gpu is caught here
 
```

Comment:
I think we can use `'int4_gpu' in test_api` to ensure that the previous `run_transformer_int4_gpu_win` is not affected.